controllers/activities.js

```diff
@@ -46,13 +46,24 @@ const update = async (req, res) => {
   }
 }
 
+const deleteActivity = async (req, res) => {
+  try {
+    const activity = await Activity.findByIdAndDelete(req.params.id)
+    const profile = await Profile.findById(req.user.profile)
```

Comment:
@J3NNog1 we don't need to do anything with the Profile here, this is because on our Profile model, there is no array of Activities...